src/modules/visitor-space/components/FilterMenu/FilterForm/FilterForm.module.scss

```diff
@@ -27,7 +27,7 @@
 
 .c-filter-form__body {
 	max-height: 100%;
-	overflow: visible;
+	overflow: scroll;
```

Comment:
on windows this will always show scrollbars.
![image](https://user-images.githubusercontent.com/1710840/223672746-4d12f7b3-eed1-472d-b158-404dfb032c73.png)


maybe this is an option?
```suggestion
	overflow-y: auto;
```